doc/vim-iced.txt

```diff
@@ -819,8 +819,7 @@ COMMANDS                                                    *vim-iced-commands*
 
                                                         *:IcedEvalOuterTopList*
 :IcedEvalOuterTopList
-  Work same as `<Plug>(iced_eval)<Plug>(sexp_outer_top_list)`,
-  without scrolling.
+  Evaluate topmost form that is not a comment where the cursor currently is.
```

Comment:
It may be helpful there is an examples like the following.
https://github.com/liquidz/vim-iced/blob/3.1.3/doc/vim-iced.txt#L400

Also if you don't mind, could you update the following document page?
https://github.com/liquidz/vim-iced/blob/3.1.3/doc/pages/configuration/evaluation.adoc#evaluation

Reply:
sure, I've put the example in the adoc and linked to it from vim doc